cgt_calc/main.py

```diff
@@ -768,6 +795,25 @@ def calculate_capital_gain(
                             calculation_log[spin_off.date][
                                 f"spin-off${spin_off.source}"
                             ] = [spin_off_entry]
+            if date_index in self.transfer_to_spouse_list:
+                for symbol in self.transfer_to_spouse_list[date_index]:
+                    transaction = self.transfer_to_spouse_list[date_index][symbol]
+                    current_quantity = self.portfolio[symbol].quantity
+                    current_amount = self.portfolio[symbol].amount
+                    amount = current_amount * transaction.quantity / current_quantity
+                    self.portfolio[symbol] -= Position(transaction.quantity, amount)
+                    transfer_entry = CalculationEntry(
+                        rule_type=RuleType.TRANSFER_TO_SPOUSE,
+                        quantity=transaction.quantity,
+                        amount=-amount,
+                        new_quantity=self.portfolio[symbol].quantity,
+                        new_pool_cost=self.portfolio[symbol].amount,
+                        fees=transaction.fees,
+                        allowable_cost=amount,
+                    )
+                    calculation_log[date_index][f"transfer-to-spouse${symbol}"] = [
```

Comment:
You probably only want to do this if `date_index >= tax_year_start_index`, right?